MCB-Nucleo/Core/Inc/MCB.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * MCB.h
+ *
+ *  Created on: Jun. 1, 2023
+ *      Author: kcgro
+ */
+
+#ifndef INC_MCB_H_
+#define INC_MCB_H_
+
+#include "can.h"
+
+#define CAN_DATA_LENGTH 8
+#define ADC_POLL_DELAY 10
+#define SEND_MOTOR_COMMAND_DELAY 10
+#define ADC_DEADZONE 500
+#define ADC_MAX 4096
+#define TRUE 1
+#define FALSE 0
+
+#define EVENT_FLAG_UPDATE_DELAY 5
+#define MIN_REVERSE_VELOCITY 3
+#define GET_CAN_VELOCITY_DELAY 500
+#define READ_BATTERY_SOC_DELAY 5000
+#define DELAY_MOTOR_STATE_MACHINE 10
+
+#define CRUISE_INCREMENT_VAL 1 // How much pressing the cruise up/down buttons increase or decrease the cruise velocity
+#define CRUISE_MAX 30 // Max cruise speed
+#define CRUISE_MIN 5 // Min cruise speed
```

Comment:
What are the units and how did these get chosen?

Reply:
Units are m/s, I added a comment to specify this.
Values are placeholders and need to be specified,